doc/config.toml

```diff
@@ -28,6 +28,10 @@ pygmentsUseClasses = true
   github_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
   github_repository_edit = "https://github.com/TheThingsNetwork/lorawan-stack/edit/master/doc/content"
   version = "v3.8.1"
+  [params.search]
+    enabled = true
+    api_key = "4372282a02970d8402d43457bf95d486"
```

Comment:
You'll need to source this from an environment variable. See: https://gohugo.io/getting-started/configuration/#configure-with-environment-variables

API keys should never be committed since now anyone can read it and use it for themselves. So in addition to sourcing this from ENV, you should generate a new API key and invalidate this one.

Reply:
But how do we manage our environmental variables currently? Would this be the only one?

Reply:
According to Algolia, this API key is search only and is safe to publish. They won't issue a new key.